gio-sys/src/lib.rs

```diff
@@ -8830,6 +8862,11 @@ extern "C" {
     //=========================================================================
     pub fn g_io_module_scope_flags_get_type() -> GType;
 
+    //=========================================================================
+    // GMemoryMonitorWarningLevel
+    //=========================================================================
+    pub fn g_memory_monitor_warning_level_get_type() -> GType;
```

Comment:
Should be `v2_64` or not?

Reply:
Usually get_types won't have version as no way to set in .gir file
And seems Gir in sys mode also ignores object version overrides :(

Reply:
I guess we can keep them around in -sys as they are, and only use them in the non-sys bindings if the corresponding feature flag is enabled?

And in the non-sys bindings we would then generate only the trait impls if the feature flag is enabled, and put a manual override into the `Gir.toml` for the version?

For the other types we would have to generate two `glib_wrapper!` calls. Doesn't seem impossible to solve :)

Reply:
We already have https://github.com/gtk-rs/gir/issues/839 for that, and it already linked from this PR ;)

Reply:
Ah great, so don't add the hacks that return `0` but make it fail properly when people use it (it will then fail with a linker error instead of exploding in incomprehensible ways at runtime).

Reply:
Seems good to me now except for this one

Reply:
Still not happy with this one :)

Reply:
Seems currently no way to change version here, I can't even ignore this function :(

Reply:
What's the problem with just keeping it around then and not having the dummy implementations? That's what we're doing since forever and what caused me to create gtk-rs/gir#839.

Reply:
No problem with this function as it added at same time as enum